docs/topics/mta-web-creating-custom-migration-targets.adoc

```diff
@@ -6,7 +6,9 @@
 [id="mta-web-creating-custom-migration-targets_{context}"]
 = Creating custom migration targets
 
-Users with `admin` permissions can create and maintain custom rulesets (migration targets). Non-admin users then can use those migration targets for analyzing applications.
+The purpose of this feature is to enable architects or users with `admin` permissions to create and maintain custom rulesets (migration targets). Architects can upload custom rule files and assign them to various custom migration targets. The custom migration targets are then available for selecting in the analysis configuration wizard.
```

Comment:
"The purpose of this feature is to enable architects or users with `admin` permissions to create and maintain custom rulesets associated to custom migration targets"

Reply:
@rromannissen 
Changed, please see.